### PR TITLE
feat (core): add webpack config

### PR DESCRIPTION
**Pull request by Atzkdeff**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+
+module.exports = (env, argv) => ({
+    output: {
+        path: path.join(__dirname, "/dist"),
+        filename: "index.bundle.js"
+    },
+    resolve: {
+        extensions: [".js"],
```
Comment (VladislavGordeyko) [inline webpack.config.js]: It's better to fix the webpack config to work with JSX file types. You can check here - https://stackoverflow.com/questions/46169472/reactjs-js-vs-jsx

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    devServer: {
+        port: 3000,
+        liveReload: true
+    },
+    devtool: argv.mode === "development" ? "source-map" : undefined,
+    module: {
+        rules: [
+            {
+                test: /\.(js|jsx)$/,
+                exclude: /nodeModules/,
```
Comment (VladislavGordeyko) [inline webpack.config.js]: node_modules*

**Review by VladislavGordeyko — CHANGES_REQUESTED** [review]

File: webpack.config.js (hunk 1 above)
Comment (Atzkdeff) [reply webpack.config.js]: Fixed

File: webpack.config.js (hunk 2 above)
Comment (Atzkdeff) [reply webpack.config.js]: Fixed

**Review by VladislavGordeyko — APPROVED** [review]

**Pull request merged**